packages/trace-viewer/src/snapshotRenderer.ts

```diff
@@ -273,3 +274,33 @@ function snapshotScript() {
 
   return `\n(${applyPlaywrightAttributes.toString()})()`;
 }
+
+const schemas = ['about:', 'blob:', 'data:', 'file:', 'ftp:', 'http:', 'https:', 'mailto:', 'sftp:', 'ws:', 'wss:' ];
+
+const kLegacyBlobPrefix = 'http://playwright.bloburl/#';
+
+export function rewriteURLForCustomProtocol(href: string): string {
+  // Legacy support, we used to prepend this to blobs, strip it away.
+  if (href.startsWith(kLegacyBlobPrefix))
+    href = href.substring(kLegacyBlobPrefix.length);
+
+  try {
+    const url = new URL(href);
+    // Sanitize URL.
+    if (url.protocol === 'javascript:')
+      return 'javascript:void(0)';
+
+    // Pass through if possible.
+    const isBlob = url.protocol === 'blob:';
```

Comment:
Can just  remove "blob" from the list, and everything should work?

Reply:
Done.